Thinktecture.Relay.Server/Http/ActionFilters/NoCacheAttribute.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Net.Http.Headers;
+using System.Web.Http.Filters;
+
+namespace Thinktecture.Relay.Server.Http.ActionFilters
```

Comment:
This should be named `Filters` instead, because the Frameworks namespace is `Filters`, too.